### PR TITLE
Fix postgres timestamp type

### PR DESCRIPTION
**Pull request by khorchanov**
May I suggest an easy non breaking change to fix this #275

### CODE REVIEW

File: packages/apalis-sql/src/postgres.rs [hunk 1]
```diff
@@ -681,6 +682,23 @@ mod tests {
         cleanup(storage, &worker_id).await;
     }
 
+    #[tokio::test]
+    async fn test_schedule_job() {
+        let mut storage = setup().await;
+        let time = Utc::now() + Duration::from_secs(1);
```
Comment (geofmureithi) [inline packages/apalis-sql/src/postgres.rs:688]: Is it possible to add a larger number of seconds, ensure its not started, sleep then check again?

**Review by geofmureithi — COMMENTED** [review]
Also check the cargo check issues.

**Comment by geofmureithi** [comment]
Let me know if you are able to improve the tests, to check if scheduling works as expected.

File: packages/apalis-sql/src/postgres.rs (hunk 1 above)
Comment (khorchanov) [reply packages/apalis-sql/src/postgres.rs]: What about now ?

**Comment by geofmureithi** [comment]
@khorchanov Seems the tests are not passing still.

**Pull request closed without merging**